### PR TITLE
Add ability to stop axis motor from axis

### PR DESCRIPTION
**Pull request by JamesKingWork**
https://github.com/ISISComputingGroup/IBEX/issues/4579

### CODE REVIEW

File: axisApp/Db/axis_base.template [hunk 1]
```diff
@@ -123,7 +123,9 @@ record(stringin, "$(P)$(AXIS):MOTOR") {
 
 record(bo, "$(P)$(AXIS):STOP") {
   field(DESC, "Stop the axis motor")
-  field(OUT, "$(P)$(mAXIS).STOP 1")
+  field(OUT, "$(P)$(mAXIS).STOP")
```
Comment (FreddieAkeroyd) [inline axisApp/Db/axis_base.template]: Do you need PP ?

Comment (JamesKingWork) [reply axisApp/Db/axis_base.template]: It doesn't seem to be a problem when testing, and it's talking to a separate IOC, I though PP was ignored when talking to another IOC?

Comment (JamesKingWork) [reply axisApp/Db/axis_base.template]: It turns out using PP here does fix a strange issue where if you stop the axis and then write a new setpoint the setpoint is ignored (but if you then set the setpoint again the motor moves).

**Comment by JamesKingWork** [comment]
This is actually not required as we already have an alias to the motor ioc, which I had not realised

File: axisApp/Db/axis_base.template (hunk 1 above)
Comment (FreddieAkeroyd) [reply axisApp/Db/axis_base.template]: With a separate IOC PP is ignored and the effective "pp" of the destination field is used, but there is no harm including it and it protects you if the db was used for a motor in the same IOC.

**Pull request closed without merging**